### PR TITLE
[Release] Update changelog for 3.11.1

### PR DESCRIPTION
**Pull request by brianlheim**
## Purpose and Motivation

there were 93 PRs merged for this patch release! including 24 documentation PRs.

when a PR was cherry-picked, i used the original PR number.

## Types of changes

- Documentation

## To-do list

- [x] Code is tested
- [x] All tests are passing
- [x] Updated documentation
- [x] This PR is ready for review

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -12,8 +12,157 @@ Release dates of 3.x versions:
...
+
+Both servers now warn about an issue where Windows Defender may delay booting by ~1 minute ([#4984](https://github.com/supercollider/supercollider/pull/4984))
+
+supernova now uses FFTW as its FFT library on macOS to solve a thread safety issue ([#4583](https://github.com/supercollider/supercollider/pull/4583))
+
+scsynth now supports non-ASCII device names on the command line on Windows ([#4479](https://github.com/supercollider/supercollider/pull/4479))
+
+### UGens: Fixed
+
+TWindex learned to respond to multiple triggers in a single control block ([#5002](https://github.com/supercollider/supercollider/pull/5002))
```
Comment (jrsurge) [inline CHANGELOG.md:80]: Is this a breaking change?

File: CHANGELOG.md [hunk 2]
```diff
@@ -12,8 +12,157 @@ Release dates of 3.x versions:
...
+
+Passing a nil spec to NamedControl no longer overwrites a spec in SynthDef metadata ([#4817](https://github.com/supercollider/supercollider/pull/4817))
+
+Fixed a deprecation that broke non-GUI builds ([#4875](https://github.com/supercollider/supercollider/pull/4875))
+
+String:-runInTerminal works again on macOS ([#4880](https://github.com/supercollider/supercollider/pull/4880))
+
+UnitTest:-assertFloatEquals's formatting was fixed ([#4946](https://github.com/supercollider/supercollider/pull/4946))
+
+String:-splitext no longer interprets a period in a path component as the start of a file extension ([#4960](https://github.com/supercollider/supercollider/pull/4960))
```
Comment (jrsurge) [inline CHANGELOG.md:130]: I'm not sure anything relies on the old behaviour, but is this technically a breaking change?

File: CHANGELOG.md [hunk 3]
```diff
@@ -12,8 +12,157 @@ Release dates of 3.x versions:
...
+
+Added a new script to help automate the release process ([#4987](https://github.com/supercollider/supercollider/pull/4987), [#5114](https://github.com/supercollider/supercollider/pull/5114))
+
+### General: Changed
+
+Building sclang with readline now requires ncurses; this is to fix build errors on some Linux systems where libreadline is not fully linked ([#4900](https://github.com/supercollider/supercollider/pull/4900))
+
+The 'render all help' CMake target no longer includes extensions ([#5089](https://github.com/supercollider/supercollider/pull/5089))
+
+Changed the disableBufferAliasing argument of argument of the plugin interface's registerUnit() from int to bool ([#4710](https://github.com/supercollider/supercollider/pull/4710))
```
Comment (jrsurge) [inline CHANGELOG.md:42]: `argument of` repeated

File: CHANGELOG.md (hunk 2 above)
Comment (brianlheim) [reply CHANGELOG.md:130]: the old behavior was wrong according to spec. IMO we shouldn't label every bugfix a breaking change, or the term becomes meaningless.

**Comment by brianlheim** [comment]
@jrsurge thanks for the review, updated! i renamed the "UGens: added" to "UGens: changed", let me know if you want something else instead

File: CHANGELOG.md (hunk 2 above)
Comment (jrsurge) [reply CHANGELOG.md:130]: agreed - it's only breaking if user code can reasonably rely on it, which it can't in this case, just wanted to double check

**Pull request merged**